src/lib.rs

```diff
@@ -1,2 +1,396 @@
 extern crate libc;
-include!(concat!(env!("CARGO_MANIFEST_DIR"), "/src/arrayfire.rs"));
\ No newline at end of file
+
+use std::fmt;
+use libc::c_void;
+use libc::c_int;
+use libc::c_uint;
+use libc::c_double;
+use libc::c_longlong;
+
+use std::ops::Index;
+use std::ops::Add;
+
+#[link(name="afcpu")]
```

Comment:
Sure, i was trying out different things with cfg attributes almost entire evening yesterday, which is why i was surprised when you suggested `af*` and i wondered if it was that simple! while i was trying all other things.